package/MDAnalysis/analysis/density.py

```diff
@@ -497,12 +495,53 @@ def __repr__(self):
             grid_type = 'histogram'
         return '<Density ' + grid_type + ' with ' + str(self.grid.shape) + ' bins>'
 
+def _set_user_grid(gridcenter, xdim, ydim, zdim, smin, smax):
+    """Helper function to set the grid dimensions to user defined values
+       
+       Called by : density_from_Universe 
+       Condition : gridcenter must be defined
+
+       Parameters
+       ----------
+       gridcenter : numpy array containing the x,y,z of the box center (np.float32)
+       x/y/zdim :   box edge lengths in each dimension
```

Comment:
does not look like numpy-style docs (although this is an internal function so it's not a big deal)